docs/UserGuide.md

```diff
@@ -1,186 +1,175 @@
+   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will display help. 
 5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
+   * **`add`** : adds a task (e.g. `add buy some milk  d/tomorrow p/high`)
+   * **`edit`** : edits an existing task (e.g. `edit 1 buy some milk and cheese p/low`)
+   * **`kill`** : deletes a task in the current list (e.g. `kill 1`)
+   * **`quit`** : quits the app (e.g. `quit`)
 6. Refer to the [Features](#features) section below for details of each command.<br>
 
 
-## 2. Features
+## Features
 
 > **Command Format**
->
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
 
-### 2.1. Viewing help : `help`
-
+#### Viewing help : `help / h`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
+ 
+#### Adding a task: `add / a`
+Adds a task to the todo list<br>
+Format: `add TASK [d/DEADLINE] [p/PRIORITY] [t/TAGNAME]` 
 
-### 2.2. Adding a person: `add`
-
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
-
-> Persons can have any number of tags (including 0)
-
-Examples:
-
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
-
-### 2.3. Listing all persons : `list`
-
-Shows a list of all persons in the address book.<br>
-Format: `list`
+> Tasks can have any number of tags (including 0)
 
-### 2.4. Editing a person : `edit`
+Examples: 
+* `add Buy donuts d/tomorrow p/high t/work`
+* `add Buy milk d/Friday p/low t/home`
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+#### Edit a task: `edit / e`
+> Edit a particular task according to the index.
+> Format: `edit INDEX [TASK NAME] [d/DEADLINE] [p/PRIORITY] [t/TAGNAME]`
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
-    The index **must be a positive integer** 1, 2, 3, ...
-> * At least one of the optional fields must be provided.
-> * Existing values will be updated to the input values.
-> * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+Examples: 
+* `edit 5 Sell drugs d/Saturday p/high t/gang streets kids`
+* `edit 5 Buy drugs`
+* `edit 5 t/Exercise Personal d/31/12/2013`
```

Comment:
As your parameters are optional, more thinking can be placed for this `edit` function.
Maybe you can explain more, like what will happen to the existing values that are not specified in a particular `edit` command.
Eg. edit 2 assignment
- What will happen to the dates that were earlier on but not included now?
- Or what if I intend to leave out a start time to convert it from a task that only have deadline, etc.